sale_commission/readme/CONFIGURE.rst

```diff
@@ -21,6 +21,8 @@ For adding commissions:
 
    * **Invoice Based**: Commissions are settled when the invoice is issued.
    * **Payment Based**: Commissions are settled when the invoice is paid.
+   * **Based Payment amount** : Commission are settled when at least one 
+     payment is registered. It allows to settle partial commissions.
```

Comment:
* **Payment Amount Based** : Commissions are settled even when invoice is not fully paid. It allows to settle partial commissions, proportionally to payment received.